src/mono/wasm/Wasm.Build.Tests/PInvokeTableGeneratorTests.cs

```diff
@@ -884,5 +884,80 @@ public static unsafe int Main(string[] argv)
         [BuildAndRun(host: RunHost.Chrome, aot: false)]
         public void EnsureWasmAbiRulesAreFollowedInInterpreter(BuildArgs buildArgs, RunHost host, string id) =>
             EnsureWasmAbiRulesAreFollowed(buildArgs, host, id);
+
+        [Theory]
+        [BuildAndRun(host: RunHost.Chrome, aot: true)]
```

Comment:
```suggestion
        [BuildAndRun(host: RunHost.Chrome, aot: false)]
        [BuildAndRun(host: RunHost.Chrome, aot: true)]
```
For later when the current CI will be green.

Reply:
For now I will only run ` aot: false` case. There are issue with `aot` mode when we publish the app.

Reply:
Exactly. AOT case can be solved in a separate PR.